scripts/run-e2e.ts

```diff
@@ -134,6 +141,28 @@ const addRequiredDeps = async ({ cwd, additionalDeps }: Options) => {
   }
 };
 
+const addTypescript = async ({ cwd }: Options) => {
```

Comment:
Maybe it's a good idea to pass a typescript version and use latest as default? To allow more flexibility for testing different scenarios

Reply:
I thought about this as well but to keep things simple for now I think it's best to omit this for now